dependencies.yaml

```
@@ -108,6 +108,20 @@ dependencies:
       match: REVISION:\ '\d+'
 
   # go-runner
+  - name: "registry.k8s.io/build-image/go-runner (go1.22-bullseye)"
+    version: v2.3.1-go1.22rc1-bookworm.0
+    refPaths:
+    - path: images/build/go-runner/variants.yaml
+      match: v((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)
+
+  - name: "registry.k8s.io/build-image/go-runner: image revision (go1.22-bullseye)"
```

Shouldn't this be `bookworm`?
```suggestion
  - name: "registry.k8s.io/build-image/go-runner: image revision (go1.22-bookworm)"
```

---

thanks for all the catches @xmudrii